actors/migration/nv15/miner.go

```diff
@@ -216,7 +216,8 @@ func (m minerMigrator) migrateDeadlines(ctx context.Context, store adt.Store, de
 
 	var outDeadlines miner7.Deadlines
 	for i, c := range inDeadlines.Due {
-		if c == m.emptyDeadlineV6 && sectors == m.emptySectorsV7 {
+		if c == m.emptyDeadlineV6 {
+			// empty deadlines don't get sectors snapshots
```

Comment:
And dead deadlines? Or deadlines with no snapshotted proofs? I think we need a to check if there are snapshotted proofs here. If not, we skip.

Reply:
I convinced myself that anything except a fully empty deadline would be `Process`ed in 24 hours, at which point it would have its sectors zeroed out. 

Maybe that isn't true? And if not, shall we just check for `Live`ness here, and Snapshot sectors iff it's alive?

Reply:
Deadlines can be _dead_ but non-empty.

Simper than checking liveness: snapshot if and only if there's a proof snapshot.

Reply:
Yeah, makes sense.